### PR TITLE
feat(clusters): add "Which should I chose"

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-1012

2024-02-22-144204_1700x750_scrot

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Truth can only be found in one place: the code

File: libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx [hunk 1]
```diff
@@ -80,8 +81,36 @@ export function StepGeneral(props: StepGeneralProps) {
             }}
             render={({ field }) => (
               <BlockContent title="Installation type">
+                <Popover.Root>
+                  <Popover.Trigger>
+                    <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium mt-2">
+                      Which should I chose? <Icon className="text-xs" name={IconAwesomeEnum.CIRCLE_QUESTION} />
```
Comment (RemiBonnet) [inline libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx:87]: not sur but it's `choose` no?

File: libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx [hunk 2]
```diff
@@ -80,8 +81,36 @@ export function StepGeneral(props: StepGeneralProps) {
             }}
             render={({ field }) => (
               <BlockContent title="Installation type">
+                <Popover.Root>
+                  <Popover.Trigger>
+                    <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium mt-2">
```
Comment (RemiBonnet) [inline libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx:86]:
```suggestion
                    <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium">
```

**Review by RemiBonnet — CHANGES_REQUESTED** [review]

File: libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx (hunk 1 above)
Comment (ctjhoa) [reply libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx]: Yes you're right it's a typo from the figma
https://github.com/Qovery/console/pull/1194/commits/69f36912034cfe937ef005bcf8be85eb9e73712b

File: libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx (hunk 2 above)
Comment (ctjhoa) [reply libs/pages/clusters/src/lib/ui/page-clusters-create/step-general/step-general.tsx]: https://github.com/Qovery/console/pull/1194/commits/69f36912034cfe937ef005bcf8be85eb9e73712b

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**